src/main/java/htsjdk/tribble/AsciiFeatureCodec.java

```diff
@@ -49,14 +50,19 @@ public boolean isDone(final LineIterator lineIterator) {
     }
 
     @Override
-    public LocationAware makeIndexableSourceFromStream(final InputStream bufferedInputStream) {
-        final PositionalBufferedStream pbs;
-        if (bufferedInputStream instanceof PositionalBufferedStream) {
-            pbs = (PositionalBufferedStream) bufferedInputStream;
+    public LocationAware makeIndexableSourceFromStream(final InputStream inputStream) {
+        if (inputStream instanceof BlockCompressedInputStream) {
```

Comment:
Since we had to duplicate this code in GATK anyway, I added a "from" method per this suggestion.